src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.cs

```diff
@@ -6,6 +6,7 @@
 using Uno.Extensions;
 using Uno.Extensions.Specialized;
 using Windows.UI.Xaml.Controls.Primitives;
+using Windows.Foundation.Collections;
```

Comment:
I fail to see why you need to add this namespace.

Reply:
Ah right, sorry. I had changed the OnItemChanged(object e) to OnItemChanged(IVectorChangedEventArgs e) which comes from windows.foundation.collections, but that broke things because it's UWP standard (oops). Just forgot to change it back.